js/damage.js

```diff
@@ -342,7 +342,7 @@ function getDamageResult(attacker, defender, move, field) {
             (attacker.ability === "Strong Jaw" && move.isBite)) {
         bpMods.push(0x1800);
         description.attackerAbility = attacker.ability;
-    } else if (attacker.ability === "Tough Claws" && (move.makesContact && attacker.Ability !== "Long Reach")) {
+    } else if (attacker.ability === "Tough Claws" && (move.makesContact && attacker.ability !== "Long Reach")) {
```

Comment:
Not your mistake, but the check for Long Reach is unneeded here. If the attacker's ability is Tough Claws it can't also be Long Reach.